{{cookiecutter.project_repo_name}}/environment.yml

```diff
@@ -1,10 +1,11 @@
 name: {{ cookiecutter.project_slug }}
 channels:
-#- conda-forge
+- conda-forge
 - defaults
 dependencies:
 - pip
-- pywps>=4.2
```

Comment:
>=4.2,<4.3?  I wonder if just '=4.2' will allow 4.2.4.

Reply:
I think `~=4.2` will have the same effect.

Reply:
I have add pywps version range:
```